### DISCUSSION TITLE
Reset Either

### DISCUSSION THREAD

**Discussion by RohitSrinivasRG** [Mindgrove-Technologies/mindgrove-bsc#3] [General]
@mounakrishna99 Did we make any changes to Reset Either on suggestion from IMEC?

**Comment by mounakrishna99** [comment]
I don't think so. Steven said there won't be any glitches, so i didnt touch it.
Can you once confirm by checking that file ResetEither.v in mindgrove-bsc and open-bsc?

**Comment by mounakrishna99** [comment]
I checked it. There are no changes in ResetEither.v file.
